composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "wmde/fundraising-frontend-content",
     "description": "i18n for FundraisingFrontend",
-    "license": "GPL-2.0+",
+    "license": "GPL-2.0-or-later",
```

Comment:
This fixes the problem, but I'd still leave https://phabricator.wikimedia.org/T184994 open for @KaiNissen to decide if we want "CC-BY-SA-4.0" instead (since this repo is 99% content).